components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/config/ConfigurationFacade.java

```diff
@@ -59,12 +69,90 @@ public static ConfigurationFacade getInstance() {
         return instance;
     }
 
+    /**
+     * Returns the sequence config with given parameters.
+     * @param reqType
+     * @param relyingParty
+     * @param tenantDomain
+     * @return
+     * @throws FrameworkException
```

Comment:
Missing parameter descriptions. Please remove the parameter since this method is deprecated. Otherwise, these pops up as yellow lines in IntelliJ IDEA!!